runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -137,17 +139,36 @@ MotherlodeConfig getConfig(ConfigManager configManager)
 		return Arrays.asList(overlay, rocksOverlay, motherlodeSackOverlay, motherlodeGemOverlay);
 	}
 
+	private void refreshSackValues()
+	{
+		curSackSize = client.getVar(Varbits.SACK_NUMBER);
+		boolean sackUpgraded = client.getVar(Varbits.SACK_UPGRADED) == 1;
+		maxSackSize = sackUpgraded ? SACK_LARGE_SIZE : SACK_SIZE;
+	}
+
 	@Override
 	protected void startUp()
 	{
 		inMlm = checkInMlm();
+
+		if (inMlm)
+		{
+			refreshSackValues();
+		}
 	}
 
 	@Override
 	protected void shutDown() throws Exception
 	{
 		veins.clear();
 		rocks.clear();
+
+		Widget sack = client.getWidget(WidgetInfo.MOTHERLODE_MINE);
```

Comment:
This should also probably reset the MLM session

Reply:
Added a reset function to session that resets the session stats